### ISSUE TITLE
Refresh doesn't work

### ISSUE THREAD

**Issue by scgf** [leinelissen/jellyfin-audio-player#86]
Today I added a new album to my media folder on my NAS and went to Jellyfin Dasboard to scan my library. The album showed up in the Jellyfin web UI, and also in Finamp. It does not show in jellyfin-audio-player no matter how many times I pull down the page in iOS to refresh.

Is there something else I need to do, or is this a bug?

**Comment by leinelissen** [comment]
This depends on how your album is tagged in Jellyfin. We retrieve albums directly from the Jellyfin API with a filter on the item types. We only retrieve `MusicAlbum `. If your album is tagged with this item type, it should be showing up in the app. I think Finamp might also include `Audio` as an item type. If you check the requests to the API in the Jellyfin web interface, it should be showing the item type as well. Could you check that out?
For reference, the filters are found here: 
https://github.com/leinelissen/jellyfin-audio-player/blob/cb792c8dd40f244c7841b109a0b062ba4a54498d/src/utility/JellyfinApi.ts#L69

**Comment by scgf** [comment]
Thank you. You said "If you check the requests to the API in the Jellyfin web interface, it should be showing the item type as well.". Where do I look for this information? I go to Dashboard, read the logs - but I can see no reference to tags. If I select API Keys, it's empty. The album in question shows up everywhere else, including my BlueSound media player. Plex sees it as well as do all my Plex clients. jellyfin-audio-player is the only place where it doesn't show.

**Comment by leinelissen** [comment]
Never mind this, I've checked the Jellyfin web interface and it applies the same filters as Jellyfin Audio Player. Can you double-check that the album in question shows up under the Music → Albums page in Jellyfin, and that you're logged into the exact same webserver with the exact same user? And is it just the one album not showing up, or is it that all albums are missing?

**Comment by scgf** [comment]
Yes, the album shows up in Jellyfin/Music/Albums but will not show up in jellyfin-audio-player. All my other albums show up correctly, just not this latest one. The webserver details are the same for all clients.

**Comment by leinelissen** [comment]
Okay, thanks for checking. One last thing I just want to double check is that this doesn't have anything to do with caching. If you go to the Settings → Cache and then press Empty Cache, all cache should be removed. If you then refresh the album view, does the album pop up?

**Comment by weiyideai520** [comment]
I can't refresh the library either. Even if I uninstall the app and reinstall it, it doesn't work.
